FSJ_django20_project/FSJ/models_award.py

```diff
@@ -0,0 +1,33 @@
+#from django.db import models
+import uuid
+from django.utils.translation import gettext_lazy as _
+from django.db import models
+import datetime
+
+class Award(models.Model):
+	FIRST = 1
+	SECOND = 2
+	THIRD = 3
+	FOURTH = 4
+	FIFTH = 5
+	YEAR_CHOICES = (
+		(FIRST, _('First')),
+		(SECOND, _('Second')),
+		(THIRD, _('Third')),
+		(FOURTH, _('Fourth')),
+		(FIFTH, _('Fifth'))
+	)
+	awardid = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+	award_name = models.TextField(verbose_name = _("Award Name"))
+	description = models.TextField(verbose_name = _("Description"))
+	value = models.PositiveIntegerField(verbose_name = _("Value"))
```

Comment:
Not all awards are numerical as confirmed in a meeting with the clients (a few are some other type of award).

Reply:
I'll replace value with a TextField instead